sympy/physics/optics/utils.py

```diff
@@ -171,6 +175,68 @@ def refraction_angle(incident, medium1, medium2, normal=None, plane=None):
         return Ray3D(intersection_pt, direction_ratio=drs)
 
 
+def fresnel_coefficients(angle_of_incidence, medium1, medium2):
+    # TODO documentation + examples/doctests + tests 
+    """
+    This function uses Fresnel equations to calculate reflection and 
+    transmission coefficients. Those are obtained for both polarisations
+    when the electric field vector is in the plane of incidence (labelled 'p')
+    and when the electric field vector is perpendicular to the plane of 
+    incidence (labelled 's'). There four real coefficients unless the incident
+    ray reflects in total internal in which case there are two complex ones.
+    Angle of incidence is the angle between the incident ray and the surface 
+    normal. `medium1` and `medium2` can be `Medium` or any sympifiable object.
```

Comment:
This is far from obvious, but a single back-tick is actually LaTeX in the documentation. You probably want to use two back-ticks.